pynamodb/indexes.py

```diff
@@ -141,7 +141,7 @@ def _hash_key_attribute(cls):
                 return attr_cls
 
     @classmethod
-    def _get_schema(cls) -> Dict:
+    def _get_schema(cls, add_provisioned_throughput: bool) -> Dict:
```

Comment:
nit - like to make such params that are primitive type and have no implied/natural order (e.g. not `pow(base, exp)`) to be kwargs-only
```suggestion
    def _get_schema(cls, *, add_provisioned_throughput: bool) -> Dict:
```